packages/linux-kernel/linux-kernel.pacscript

```diff
@@ -0,0 +1,5 @@
+name="linux-kernel"
+version="6.0"
+pacdeps=("linux-headers-deb" "linux-headers-generic-deb" "linux-image-unsigned-deb" "linux-modules-deb")
+description="Provides the latest Linux kernel via the Ubuntu mainline repositories"
+maintainer="Chadano <chadano@llamawebdesign.co.uk>"
```

Comment:
you must include a `url` variable. It could just be an empty file, but it must be included.

Reply:
so just
url=""
prepare=""
build=""